### DISCUSSION TITLE
Multiple error status fields via MQTT-Hassio?

### DISCUSSION THREAD

**Discussion by Commifreak** [john30/ebusd#940] [Q&A] [answered]
I setup latest devel version with latest hassio.cfg. I removed the `filter-name` field for testing to get all fields.

I noticed multiple error status fields with same name. Could anyone explain, why there are multiple fields with same name?

![grafik](https://github.com/john30/ebusd/assets/87267/28567769-5ecb-40bb-a837-ed381701da89)

![grafik](https://github.com/john30/ebusd/assets/87267/73659905-2adc-4cf9-b2eb-a508e9b77cb1)

**Comment by Commifreak** [comment]
No one?

**Reply by Koky05** [reply, marked as answer]
It is totoaly normal. If your sensor contain multiple values it is divided into multiple sensors with same name.
For example I have bai/Status01 with five values (circuite1 temperature, circuite2 temperature, pump state, hot water temperature, status of heating system). In HA it is shown as five separate sensors Status01_temp, Status01_temp1, ...
